runtime/src/common/crypto/x25519.rs

```diff
@@ -61,9 +79,9 @@ impl cbor::Decode for PrivateKey {
 
     fn try_from_cbor_value(value: cbor::Value) -> Result<Self, cbor::DecodeError> {
         let mut bytes: [u8; PRIVATE_KEY_LENGTH] = cbor::Decode::try_from_cbor_value(value)?;
-        let pk = PrivateKey(x25519_dalek::StaticSecret::from(bytes));
+        let sk = PrivateKey(x25519_dalek::StaticSecret::from(bytes));
```

Comment:
I named this `sk` to distinguish the secret key from the public key `pk`. I blame Diffie and Hellman for this problem, as they should have forbidden naming secret key private key :smiley: